packages/paste-core/components/input-box/src/InputBoxContext.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import type {Variants} from './types';
+
+interface InputBoxContextData {
+  disabled: boolean;
+  readOnly: boolean;
+  variant: Variants;
+}
+export const InputBoxContext = React.createContext<InputBoxContextData | null>(null);
+
+export const useInputBoxContext = (): InputBoxContextData => {
+  const context = React.useContext(InputBoxContext);
+  if (!context) {
+    throw new Error('useInputBoxContext must be used within a <InputBox> component.');
```

Comment:
suggestion: this error confused me bc `useInputBoxContext` is also used in `<InputCheveronWrapper>`. I think it'd be more clear if it read "... within an `InputBox` _package_ component."

Reply:
Interesting. This needs to specifically be used within the `InputBox` component, not any component in the package. It's being used in _our_ code in InputChevronWrapper, but the error is for consumers to realize they can't use `InputChevronWrapper` outside of the `InputBox`. Does that make sense?

Reply:
Ah okay. So if they try to use `InputChevronWrapper` inside a Button for example, they'd get this error? If so, I don't think a consumer would understand where `useInputBoxContent` is coming into play. Maybe referencing `InputChevronWrapper` where `useInputBoxContent` is in this message would be more clear?